src/misc.rs

```diff
@@ -43,13 +43,20 @@ impl LintPass for MiscPass {
                     // an enum is extended. So we only consider cases where a `_` wildcard is used
                     if arms[1].pats[0].node == PatWild(PatWildSingle) && 
                             arms[0].pats.len() == 1 {
+                        let body_code = snippet(cx, arms[0].body.span, "..");
+                        let suggestion = if let ExprBlock(_) = arms[0].body.node {
+                            body_code.into_owned()
+                        } else {
+                            format!("{{ {} }}", body_code)
```

Comment:
Hm, I'm not sure how to do that. `snippet` takes a span, so it will extract some part from the original code, right?